src/gtest/test_foundersreward.cpp

```diff
@@ -101,11 +101,11 @@ TEST(founders_reward_test, general) {
     CChainParams params = Params();
     
     // First testnet reward:
-    // address = t2UNzUUx8mWBCRYPRezvA363EYXyEpHokyi
-    // script = OP_HASH160 ef775f1f997f122a062fff1a2d7443abd1f9c642 OP_EQUAL
-    // raw script = a914ef775f1f997f122a062fff1a2d7443abd1f9c64287
-    EXPECT_EQ(params.GetFoundersRewardScriptAtHeight(1), ParseHex("a914ef775f1f997f122a062fff1a2d7443abd1f9c64287"));
-    EXPECT_EQ(params.GetFoundersRewardAddressAtHeight(1), "t2UNzUUx8mWBCRYPRezvA363EYXyEpHokyi");
+    // address = t2Mcc7khmWtviJKZLn9XyqPcs2zfQDn5K4X
```

Comment:
Done.  Added check for 53126 and 53127 to confirm addresses change at that point. Force pushed to keep commits clean.